osu.Game.Rulesets.Osu/Skinning/LegacyMainCirclePiece.cs

```diff
@@ -45,12 +48,27 @@ private void load(DrawableHitObject drawableObject)
 
             InternalChildren = new Drawable[]
             {
-                hitCircleSprite = new Sprite
+                circlePieces = new Container<Sprite>
```

Comment:
I'd just call this `hitCircleContainer` or something probably? Named a bit weird right now.

Reply:
wasn't sure about this name, maybe `circleSprites` would do?

Reply:
Well seeing as both contained drawables are prefixed with `hitCircle` that's what I proposed. I mostly suggested a change as this screwed me up reading the other comment (here a piece is the *background*, and there the piece is the *text*...?)